examples/hello_world_with_lwt/hello_world_with_lwt.ml

```diff
@@ -0,0 +1,97 @@
+(* This example assumes that a Mysql database 'test' exists for user 'root'.
+ * Moreover, a table 'users' defined as follows is also present in the DB:
+ *
+ * CREATE TABLE users
+ *     (
+ *     id    INT NOT NULL,
+ *     name  TEXT NOT NULL,
+ *     phone TEXT NULL,
+ *     PRIMARY KEY (id)
+ *     );
+ *)
+
+open Mysql_with_lwt
+
+(** {1 Database queries using the Ppx_mysql syntax extension} *)
+
+let get_users =
+  [%mysql Select_all "SELECT @int32{id}, @string{name}, @string?{phone} FROM users"]
+
+
+let get_user =
+  [%mysql
+    Select_one
+      "SELECT @int32{id}, @string{name}, @string?{phone} FROM users WHERE id = %int32{id}"]
+
+
+let insert_user =
+  [%mysql
+    Execute
+      "INSERT INTO users (id, name, phone) VALUES (%int32{id}, %string{name}, \
+       %string?{phone})"]
+
+
+let update_user =
+  [%mysql
+    Execute
+      "UPDATE users SET name = %string{name}, phone = %string?{phone} WHERE id = \
+       %int32{id}"]
+
+
+let delete_user = [%mysql Execute "DELETE FROM users WHERE id = %int32{id}"]
+
+(** {1 Main functions and values} *)
+
+let print_user (id, name, phone) =
+  Lwt_io.printf
+    "%ld -> %s (phone: %s)\n"
+    id
+    name
+    ( match phone with
```

Comment:
Did you run this through `ocamlformat`? It seems a bit strange that it does a space between `(` and `match`.

Reply:
Yeah, that is OCamlformat's doing.

Reply:
Are you sure it is using the configuration file? I think I made it not do that.

Reply:
Yes, it is using the config file.